tests/PHPUnit/Integration/Segment/SegmentUnavailableTest.php

```diff
@@ -0,0 +1,186 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Tests\Integration\Segment;
+
+use Piwik\ArchiveProcessor\Rules;
+use Piwik\Cache;
+use Piwik\Date;
+use Piwik\Option;
+use Piwik\Plugins\Live\SystemSettings;
+use Piwik\Tests\Framework\Mock\FakeAccess;
+use Piwik\Tests\Framework\TestCase\IntegrationTestCase;
+use Piwik\Plugins\SegmentEditor\API;
+use Piwik\Plugins\SitesManager\API as APISitesManager;
+
+/**
+ * @group Segment
+ * @group SegmentUnavailableTest
+ */
+class SegmentUnavailableTest extends IntegrationTestCase
+{
+
+    protected $idSite = 1;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        // Setup test site
+        Date::$now = strtotime('2020-03-01 00:00:00');
+        FakeAccess::setIdSitesView([1, 2]);
+        FakeAccess::setIdSitesAdmin([3, 4]);
+        FakeAccess::$superUser = true;
+        FakeAccess::$superUserLogin = 'superusertest';
+        APISitesManager::getInstance()->addSite('test', 'http://example.org');
+
+        Option::set(Rules::OPTION_BROWSER_TRIGGER_ARCHIVING, 0);
+    }
+
+    public function tearDown() : void
+    {
+        parent::tearDown();
+        Option::set(Rules::OPTION_BROWSER_TRIGGER_ARCHIVING, 1);
```

Comment:
Should we do `Option::get` in the `setUp` method saving the value into an instance property and then restore it here to what it was? It may not matter much but feels cleaner than explicitly setting it to on for any tests coming after this test. Might cause more flakiness. No strong feelings about it either way, so happy to let it be.

Reply:
I considered doing that, but then checked what other tests were doing and decided consistency is the better part of valour :slightly_smiling_face: